src/orderFeeds/ws.order.feed.ts

```diff
@@ -118,7 +130,8 @@ export class WsNextOrder extends GetNextOrder {
 
     this.socket.on("close", async () => {
       this.logger.error(`WsConnection is closed`);
-      await setTimeout(5000);
+      clearTimeout(this.pingTimer);
+      await setTimeoutPromise(5000);
```

Comment:
What is `5000`? Please move to global constant, e.g. `RECONNECTION_DELAY`.

Why `setTimeoutPromise` instead of `setTimeout`?

Reply:
Standard `setTimeout` already supports promises. Look how you import `setTimeoutPromise`:

```
import { setTimeout as setTimeoutPromise } from "timers/promises";
```

this is not necessary to rename it. More to say, it is not necessary to import it, it would work without any import

Reply:
No we can't. And why we should? I have only two minor proposals:
- do not rename `setTimeout` to `setTimeoutPromise`
- move the magic number `5000` to constant called `RECONNECTION_DELAY`